### PR TITLE
fix: minor issue & update: pausable

### PR DESCRIPTION
**Pull request by bejaxer**

### CODE REVIEW

**Comment by bejaxer** [comment]
Result

File: bond-no-treasury/contracts/HectorBondV2NoTreasuryFTMDepository.sol [hunk 1]
```diff
@@ -1058,19 +1116,18 @@ contract HectorBondV2NoTreasuryFTMDepository is OwnableUpgradeable {
...
-        IERC20Upgradeable(_principal).safeTransfer(feeRecipient, fee);
+        IERC20Upgradeable(_principal).safeTransfer(
+            feeRecipient,
+            tokenBalances[_principal][feeRecipient]
+        );
         tokenBalances[_principal][feeRecipient] = 0;
     }
 
     function claimFund(address _principal) external {
-        uint256 fund = tokenBalances[_principal][fundRecipient];
```
Comment (michaelnguyen102) [inline bond-no-treasury/contracts/HectorBondV2NoTreasuryFTMDepository.sol:1070]: @bejaxer what's the reason to remove this check?

File: bond-no-treasury/contracts/HectorBondV2NoTreasuryFTMDepository.sol [hunk 2]
```diff
@@ -1109,14 +1166,16 @@ contract HectorBondV2NoTreasuryFTMDepository is OwnableUpgradeable {
         );
 
         uint256 fee = _amount.mul(feeBps).div(ONEinBPS);
-        tokenBalances[_principal][fundRecipient] += _amount.sub(fee);
+        tokenBalances[_principal][fundRecipient] = tokenBalances[_principal][
```
Comment (michaelnguyen102) [inline bond-no-treasury/contracts/HectorBondV2NoTreasuryFTMDepository.sol:1169]: @bejaxer isn't this the same as tokenBalances[_principal][fundRecipient] += _amount.sub(fee);?

File: bond-no-treasury/contracts/HectorBondV2NoTreasuryFTMDepository.sol [hunk 3]
```diff
@@ -1109,14 +1166,16 @@ contract HectorBondV2NoTreasuryFTMDepository is OwnableUpgradeable {
...
+            fundRecipient
+        ].add(_amount.sub(fee));
 
         if (fee > 0) {
             uint256 theLast = fee;
             for (uint256 i = 0; i < feeRecipients.length - 1; i++) {
-                tokenBalances[_principal][feeRecipients[i]] += fee
-                    .mul(feeWeightBps[i])
-                    .div(ONEinBPS);
+                tokenBalances[_principal][feeRecipients[i]] = tokenBalances[
```
Comment (michaelnguyen102) [inline bond-no-treasury/contracts/HectorBondV2NoTreasuryFTMDepository.sol:1176]: @bejaxer isn't this line doing the same thing as the deleted line?

File: bond-no-treasury/contracts/HectorBondV2NoTreasuryFTMDepository.sol [hunk 4]
```diff
@@ -626,7 +668,10 @@ interface IBondPricing {
         returns (address);
 }
 
-contract HectorBondV2NoTreasuryFTMDepository is OwnableUpgradeable {
+contract HectorBondV2NoTreasuryFTMDepository is
```
Comment (michaelnguyen102) [inline bond-no-treasury/contracts/HectorBondV2NoTreasuryFTMDepository.sol:671]: @bejaxer Will you change the contract name to BondNoTreasury? 
- Rename HectorBondV2NoTreasuryFTMDepository.sol to BondNoTreasury.sol

Comment (michaelnguyen102) [reply bond-no-treasury/contracts/HectorBondV2NoTreasuryFTMDepository.sol:671]: Add a string public variable called version = "2.0"

File: bond-no-treasury/contracts/HectorBondV2NoTreasuryFTMDepository.sol [hunk 5]
```diff
@@ -1058,19 +1116,18 @@ contract HectorBondV2NoTreasuryFTMDepository is OwnableUpgradeable {
             feeRecipient != fundRecipient,
             'can only claim fee for recipient'
         );
-
-        uint256 fee = tokenBalances[_principal][feeRecipient];
```
Comment (michaelnguyen102) [inline bond-no-treasury/contracts/HectorBondV2NoTreasuryFTMDepository.sol:1062]: @bejaxer what's the reason behind removing this check?

**Review by michaelnguyen102 — CHANGES_REQUESTED** [review]

**Review by michaelnguyen102 — APPROVED** [review]

**Pull request merged**